src/styles/main.scss

```diff
@@ -1,3 +1,81 @@
-body {
+$cell-size: 100px;
+$gap-size: 1px;
+$padding: 10px;
+$sheet-with: $cell-size * 7 + $gap-size * 6;
+
+$days-in-week: mon 1, tue 2, wed 3, thu 4, fri 5, sat 6, sun 7,;
+$days-in-month: 28, 29, 30, 31;
+
+$number-font: "Arial", sans-serif;
+$number-font-size: ($cell-size * 0.3);
+$number-font-position: ($cell-size - $number-font-size) / 2;
+
+* {
   margin: 0;
+  padding: 0;
+  animation-duration: 10s;
+}
+
+.calendar {
+  padding: $padding;
+
+  &__start-day-wed,
+  &__month-length-29 {
+    display: none;
+  }
+
+  @each $name, $num in $days-in-week {
+    &__start-day-#{$name} {
+      ~ .calendar__sheet > :first-child {
+        margin-left: $cell-size * ($num - 1) + $gap-size * ($num - 1);
+      }
+    }
+  }
+
+  @for $n from 28 to 32 {
```

Comment:
I would use a varialbe that would be more clear

Reply:
➕ , what is `n`? And why do you need 32? Calendar can have only 31 days max

Reply:
I hardcoded it because we have only 28 -> 31 days in the month, no less no more